components/Tabs/src/private/MDCItemBar.m

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
This is not a new file.
Unless we made a mistake and this file was actually created in 2017, we never change copyrights.
If we did make a mistake to our copyright, please make this change in a separate PR.